File: MonoDroid/AdMobTest/AdMobTest/AdMobHelper.cs
using System;
using Android.Runtime;
using Android.Views;

namespace AdMobTest
{
    public static class AdMobHelper
    {
        private static IntPtr _helperClass = JNIEnv.FindClass("samples/admobtest/AdMobHelper");

        public static void RegisterEmulatorAsTestDevice()
        {
            IntPtr methodId = JNIEnv.GetStaticMethodID(_helperClass, "registerEmulatorAsTestDevice", "()V");
            JNIEnv.CallStaticVoidMethod(_helperClass, methodId);
        }

        public static void RequestFreshAd(View view)
        {
            IntPtr methodId = JNIEnv.GetStaticMethodID(_helperClass, "requestFreshAd", "(Landroid/view/View;)V");
            JNIEnv.CallStaticVoidMethod(_helperClass, methodId, new JValue(view));
        }
    }
}

File: MonoDroid/AdMobTest/AdMobTest/AdMobHelper.java
package samples.admobtest;

import android.view.View;
import com.admob.android.ads.AdManager;
import com.admob.android.ads.AdView;

public class AdMobHelper
{
	private AdMobHelper() { }
	
	public static void registerEmulatorAsTestDevice()
	{
		AdManager.setTestDevices(new String[] { AdManager.TEST_EMULATOR });
	}

	public static void requestFreshAd(View view)
	{
		((AdView)view).requestFreshAd();
	}
}